app/assets/javascript/application.js

```diff
@@ -9,6 +9,33 @@ jQuery(function($) {
   });
 
   ////
+  // Add "select all"/"clear all" buttons to each select2 dropdown menu
+  $("select.select2").each(function() {
```

Comment:
Agree it would be good to get it into admin templates. I think we prefer to wait until it's needed in more than one apps to move things into it though.